auth-web/src/components/auth/AffiliatedEntityList.vue

```diff
@@ -27,9 +27,11 @@
         >
           <template v-slot:item.info="{ item }">
             <div class="meta">
-              <v-list-item-title>{{ item.name }}</v-list-item-title>
-              <v-list-item-subtitle v-if="!isNameRequest(item.corpType.code)">Incorporation Number: {{ item.businessIdentifier }}</v-list-item-subtitle>
-              <v-list-item-subtitle v-if="isNameRequest(item.corpType.code)">{{ item.corpType.desc }}: {{ item.businessIdentifier }}</v-list-item-subtitle>
+              <v-list-item-title v-if="isNumberedIncorporationApplication(item)">Numbered Benefit Company</v-list-item-title>
+              <v-list-item-title v-if="!isNumberedIncorporationApplication(item)">{{ item.name }}</v-list-item-title>
+              <v-list-item-subtitle v-if="isIncorporationNumber(item.corpType.code)">Incorporation Number: {{ item.businessIdentifier }}</v-list-item-subtitle>
```

Comment:
not necessary to change now.. but better approach will be one method which gives back the title ;something like getTitle(item) and that performs this check..
after putting this i realised this might not work.. since the third condition doesnt have any connection with first two statements..so this comment might not valid